app/assets/javascripts/pages/kube_status.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDom from "react-dom";
+import superagent from "superagent";
+import _ from "lodash";
+
+class KubeStatusPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      appStatus: this.props.appStatus
+    };
+    this.render = require("./kube_status.rt");
```

Comment:
Why use `rt`? Why not write the render method using `jsx`?

Reply:
Just followed the existing conventions